cmd/internal/client.go

```diff
@@ -44,50 +44,50 @@ func createKubernetesClient(logger logr.Logger, opts ...kube.NewOption) kubernet
 	return client
 }
 
-func CreateKyvernoClient(logger logr.Logger, opts ...kyverno.NewOption) versioned.Interface {
+func createKyvernoClient(logger logr.Logger, opts ...kyverno.NewOption) versioned.Interface {
 	logger = logger.WithName("kyverno-client")
 	logger.Info("create kyverno client...", "kubeconfig", kubeconfig, "qps", clientRateLimitQPS, "burst", clientRateLimitBurst)
 	client, err := kyverno.NewForConfig(createClientConfig(logger), opts...)
 	checkError(logger, err, "failed to create kyverno client")
 	return client
 }
 
-func CreateDynamicClient(logger logr.Logger, opts ...dyn.NewOption) dynamic.Interface {
+func createDynamicClient(logger logr.Logger, opts ...dyn.NewOption) dynamic.Interface {
 	logger = logger.WithName("dynamic-client")
 	logger.Info("create dynamic client...", "kubeconfig", kubeconfig, "qps", clientRateLimitQPS, "burst", clientRateLimitBurst)
 	client, err := dyn.NewForConfig(createClientConfig(logger), opts...)
 	checkError(logger, err, "failed to create dynamic client")
 	return client
 }
 
-func CreateMetadataClient(logger logr.Logger, opts ...meta.NewOption) metadata.Interface {
+func createMetadataClient(logger logr.Logger, opts ...meta.NewOption) metadata.Interface {
 	logger = logger.WithName("metadata-client")
 	logger.Info("create metadata client...", "kubeconfig", kubeconfig, "qps", clientRateLimitQPS, "burst", clientRateLimitBurst)
 	client, err := meta.NewForConfig(createClientConfig(logger), opts...)
 	checkError(logger, err, "failed to create metadata client")
 	return client
 }
 
-func CreateApiServerClient(logger logr.Logger, opts ...apisrv.NewOption) apiserver.Interface {
+func createApiServerClient(logger logr.Logger, opts ...apisrv.NewOption) apiserver.Interface {
 	logger = logger.WithName("apiserver-client")
 	logger.Info("create apiserver client...", "kubeconfig", kubeconfig, "qps", clientRateLimitQPS, "burst", clientRateLimitBurst)
 	client, err := apisrv.NewForConfig(createClientConfig(logger), opts...)
 	checkError(logger, err, "failed to create apiserver client")
 	return client
 }
 
+func createDClient(logger logr.Logger, ctx context.Context, dyn dynamic.Interface, kube kubernetes.Interface, resync time.Duration) dclient.Interface {
```

Comment:
The name is not obvious to understand what it returns🤔

Can we rename it to `createKyvernoDynamicClient` or something else?

Reply:
Done